en/DOItoBibTeX.md

```diff
@@ -1,15 +1,16 @@
 ---
 title: Fetching entries using the Digital Object Identifier (DOI)
 helpCategories: ["Fetching entries from the web", "... using publication identifiers"]
-outdated: true
 ---
 
 # Fetching entries using the Digital Object Identifier (DOI)
 
-To use this feature, choose **Search -&gt; Web search**, and the search interface will appear in the side pane. Select **DOI to BibTeX** in the dropdown menu.
-
-This fetcher uses <http://dx.doi.org/> to convert an DOI to a BibTeX entry.
-
-Enter the DOI in the search field and press <kdb>RETURN</kdb> or the **Fetch** button.
-The results are displayed in the [import inspection window](ImportInspectionDialog).
-In case an error occurs, it is shown in a popup.
+To use this feature, choose **BibTeX -&gt; New entry...**.
+In the lower part, there is the box "ID-based entry generator".
+In the field "ID type", "DOI" should be already selected.
+The field "ID" is focused.
+Enter the DOI here and press <kbd>Enter</kbd> to generate an entry based on the Id.
+You can also click on "Generate".
+Then, <http://dx.doi.org/> is used to convert the given DOI to a BibTeX entry.
```

Comment:
It's not `dx.doi.org` but `crossref.org` which is (currently) used to get the bibtex data. 

Not sure whether this should be documented or left out as implementation internals...